src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html

```diff
@@ -142,6 +143,27 @@
       {{name}}: ${{price}}
     </template>
   </amp-list>
+
+  <!-- E-Commerce websites tipically use carousel layout to display personalized content.
+  One way to address this is to have an `amp-list` as the src of `amp-carousel` and the amp-list endpoint
```

Comment:
suggestion: One way to address/One way to implement this is to ...'

Reply:
done